gym_kmanip/__init__.py

```diff
@@ -457,21 +397,9 @@ def vuer2mj_orn(orn: R) -> NDArray:
     kwargs={
         "mjcf_filename": TORSO_MJCF,
         "urdf_filename": TORSO_URDF,
-        "obs_list": [
-            "q_pos",  # joint positions
-            "q_vel",  # joint velocities
-            "camera/head",  # robot head camera
-            "camera/grip_l",  # left gripper camera
-            "camera/grip_r",  # right gripper camera
-        ],
-        "act_list": [
-            "eel_pos",  # left end effector position
-            "eel_orn",  # left end effector orientation
-            "eer_pos",  # right end effector position
-            "eer_orn",  # right end effector orientation
-            "grip_l",  # left gripper
-            "grip_r",  # right gripper
-        ],
+        "obs_type": ObservationType.image,
+        "cam_list": ["head", "grip_l", "grip_r"],
```

Comment:
Should we explicitly typed this as well?

Reply:
Hmm, a bit on the fence on that one. If I have to put 3 flag enums for each string and just replace strings with enums, not too different in my mind. For the other two, there are known combinations of groups of values from a list, so introducing enums to represent the known groups and abstract the code out made sense to me.

I can do it if you feel strongly - let me know.

Reply:
Strings always introduce possible bugs so I will prefer enums.

Reply:
Sure, fixed in the latest commit!